config/test/config.go

```diff
@@ -19,7 +20,24 @@ func DefaultConfigForTesting() *config.Config {
 	return cfg
 }
 
-// DefaultProfileForTesting constructs a profile with precomputed keys, only used for testing.
+// DefaultMemConfigForTesting constructs a config with precomputed keys & settings for an in memory repo & fs, only used for testing.
+func DefaultMemConfigForTesting() *config.Config {
+	cfg := DefaultConfigForTesting().Copy()
+
+	// add settings for in-mem filesystem
+	cfg.Filesystems = []qfs.Config{
+		{Type: "mem"},
+		{Type: "local"},
+		{Type: "http"},
+	}
+
+	// add settings for in mem repo
+	cfg.Repo.Type = "mem"
+
+	return cfg
+}
+
+// DefaultProfileForTesting constructs a profile with precompted keys, only used for testing.
 func DefaultProfileForTesting() *config.ProfilePod {
```

Comment:
I'd love this to move into a new `profile/test` package, but let's save that for a later PR. I'd love a 

```
TODO(b5): move this into a new profile/test package
```

while you're here 😄. Assign yourself if you're feeling bold

Reply:
got it #1715